releases/release-1.18/README.md

```diff
@@ -0,0 +1,71 @@
+# Kubernetes 1.15
+
+
+#### Links
+
+* [This document](https://git.k8s.io/sig-release/releases/release-1.18/README.md)
+* [Release Team](https://git.k8s.io/sig-release/releases/release-1.18/release_team.md)
+* [Meeting Minutes]() (join [kubernetes-sig-release@] to receive meeting invites)
+* [v1.18 Release Calendar][k8s118-calendar]
+* Contact: [#sig-release] on slack, [kubernetes-release-team@] on e-mail
+* [Internal Contact Info]() (accessible only to members of [kubernetes-release-team@])
+
+#### Tracking docs
+
+* [Enhancements Tracking Sheet]()
+* [Bug Triage Tracking Sheet]()
+* [CI Signal Report]()
+* [Retrospective Document]()
+* [kubernetes/sig-release v1.18 milestone][]
+
+#### Guides
+
+* [Targeting Issues and PRs to This Milestone](https://git.k8s.io/community/contributors/devel/sig-release/release.md)
+* [Triaging and Escalating Test Failures](https://git.k8s.io/community/contributors/devel/sig-testing/testing.md#troubleshooting-a-failure)
+
+## tl;dr
+
+The 1.16 release cycle is proposed as follows:
+
+- **Monday, January 06**: Week 1 - Release cycle begins
+- **Tuesday, January 28**: Week 4 - [Enhancements Freeze]
+- **Thursday, March 05**: Week 9 - [Code Freeze]
+- **Monday, March 16**: Week 11 - Docs must be completed and reviewed
+- **Tuesday, March 24**: Week 12 - Kubernetes v1.18.0 released
+
+
+## Timeline
+
+
+| **What** | **Who** | **Jan** | **Feb** | **Mar** | **WEEK** | **CI SIGNAL** |
```

Comment:
We probably could think about using a markdown formatter like [prettier](https://prettier.io) for tables like that to be able to read them without the need of additional rendering.